pyscriptjs/src/plugins/fetch.ts

```diff
@@ -29,7 +32,10 @@ export function calculatePaths(fetch_cfg: FetchConfig[]) {
             fetchPaths.push(from);
             const filename = to_file || from.split('/').pop();
             if (filename === '') {
-                throw new UserError(`Couldn't determine the filename from the path ${from}, supply ${to_file} parameter!`);
+                throw new UserError(
+                    UserError.ErrorCode.FETCH_PARAMETER_ERROR,
+                    `Couldn't determine the filename from the path ${from}, supply ${to_file} parameter!`
```

Comment:
I guess this is carried from a previous mistake but `${to_file} parameter` should be `"to_file" parameter` since we don't want to use string interpolation here i.e. we don't want to use the value of `to_file` inside the message, but instead communicate that you need to use this.

Reply:
Ah thanks! I'll fix it in the next push 😄